js/adapt-aboutUs.js

```diff
@@ -6,46 +6,46 @@
  */
 
 define([
-    'core/js/adapt',
-    './adapt-aboutUsView'
+  'core/js/adapt',
+  './adapt-aboutUsView'
 ], function(Adapt, AboutUsView) {
```

Comment:
we want to use imports/exports here instead of require and create a new AboutUs class, see https://github.com/HT2-Labs/adapt-contrib-reveal/blob/master/js/RevealView.js as an example.